docker-compose.yml

```diff
@@ -39,11 +39,13 @@ services:
       - ENABLE_ES=true
       - ES_SEEDS=elasticsearch
       - ES_VERSION=v7
+      - PROMETHEUS_ENDPOINT=0.0.0.0:8000
```

Comment:
Please keep these changes in the override file. I'd prefer this docker-compose.yml be as close to upstream as possible. The only reason it's not an exact copy is that you cannot remove volume configuration with an override so I have to comment those lines out here.

Reply:
hmmm we tell users to just add that as rly should be in main. will try in overrides, u got it :)